src/frontend/src/app/flow/options/options.component.ts

```diff
@@ -13,7 +16,7 @@ export class OptionsComponent {
   frankDoc: any;
   node?: Node;
   // TODO: Make types.
-  attributes!: [{ [key: string]: string }];
+  attributes!: FlowNodeAttributes;
   attributeOptions: {
```

Comment:
Could you make an attribute options type?

It should coincide with the type on the backend.

Reply:
Why does the backend need attributes?

Reply:
The attributes vary wideley in what can be in them. Things like description of default value aren't always present. By looking at the type witch is used for generating the JSON we can be sure that nothing will be left out.

Reply:
done